### PR TITLE
website extraction added

### LINKED ISSUE #92: Add Extraction of Websites To provision lambda
**Todo**
- [x] Modify the extract query to join with the websites table
- [x] Extract the website for each product
- [x] Ensure the website is correctly sent to the transform lambda

**Story**
Including the website in the data sent to transform allows the updated cost data to be extracted correctly

### PR DESCRIPTION
**Pull request by danfh00**
Adds the extraction of the website for each product from the database in the provision lambda script.
Closes #92

### CODE REVIEW

File: ETL/Provision/provision_lambda.py [hunk 1]
```diff
@@ -33,9 +33,10 @@ def read_database(conn: connection):
         raise TypeError(
             'A cursor can only be constructed from a Psycopg2 connection object')
     with get_cursor(conn) as cur:
-        cur.execute("""SELECT DISTINCT ON (product_id) product_id, product_code, url, price
+        cur.execute("""SELECT DISTINCT ON (product_id) product_id, product_code, url, price, website_name
                     FROM products
                     LEFT JOIN price_readings USING (product_id)
+                    LEFT JOIN websites USING (website_id)
                     ORDER BY product_id, reading_at DESC""")
```
Comment (BerkayDur) [inline ETL/Provision/provision_lambda.py]:
```suggestion
        cur.execute("""SELECT DISTINCT ON (product_id) product_id, product_code,
                                                                                                url, price, website_name
                    FROM products
                    LEFT JOIN price_readings USING (product_id)
                    LEFT JOIN websites USING (website_id)
                    ORDER BY product_id, reading_at DESC""")
```

**Review by BerkayDur — CHANGES_REQUESTED** [review]
Seems Good.... Other than the pylint score!!!!!

File: ETL/Provision/provision_lambda.py (hunk 1 above)
Comment (danfh00) [reply ETL/Provision/provision_lambda.py]: Fixed

**Review by BerkayDur — APPROVED** [review]
Looks good to me!!!

**Pull request merged**